docs/physics/index.rst

```diff
@@ -2,5 +2,64 @@
 
 .. _plasmapy-physics:
 
+****************************************
+The Physics Package (`plasmapy.physics`)
+****************************************
+
+.. currentmodule:: plasmapy.physics
+
+Introduction
+============
+
+`plasmapy.physics` provides theoretical formulas for calculation of physical quantities helpful for plasma physics.
+The layout of the subpackage is still in flux, but for now we have settled on providinga single `plasmapy.physics` namespace
```

Comment:
`providinga` --> `providing a`

Reply:
I just realized my vim spelling setup seems to not with .rst files! Maybe don't worry about typos for now, I'll make a pass with pycharm before merging.